src/test/kotlin/com/charleskorn/kaml/testobjects/PolymorphicTestObjects.kt

```diff
@@ -54,72 +45,66 @@ data class SealedWrapper(val element: TestSealedStructure?)
 interface SimpleInterface
 
 object SimpleNull : SimpleInterface {
-    val kSerializer: KSerializer<SimpleNull> = UnitSerializer.nullable.mapped("simpleNull", { SimpleNull }, { null })
-}
-
-data class SimpleUnit(val data: Unit) : SimpleInterface {
-    companion object {
-        val kSerializer = UnitSerializer.mapped("simpleUnit", ::SimpleUnit, SimpleUnit::data)
-    }
+    val kSerializer: KSerializer<SimpleNull> = Int.serializer().nullable.mapped("simpleNull", { SimpleNull }, { null })
```

Comment:
Is `Int.serializer()` the best choice here? This type doesn't have anything to do with `Int`s.

Reply:
You're right. I just needed a `Serializer` to make it `nullable`, because it should be `null` all the time.